src/overlay/PeerDoor.cpp

```diff
@@ -75,6 +75,12 @@ PeerDoor::handleKnock(shared_ptr<TCPPeer::SocketType> socket)
 {
     CLOG(DEBUG, "Overlay") << "PeerDoor handleKnock() @"
                            << mApp.getConfig().PEER_PORT;
+    if (!fd::disableProcessInheritance(socket->next_layer()))
```

Comment:
as you fixed this bug in a subsequent commit, I would squash the fix into this commit.

Also - you have a typo on the commit title "desscriptor"

Reply:
Sure, I'll fix this.